uniffi_bindgen/src/interface/types.rs

```diff
@@ -64,6 +64,9 @@ pub enum FFIType {
     /// The string is owned by rust and allocated on the rust heap, and must be freed by
     /// passing it to the appropriate `string_free` FFI function.
     RustError,
+    /// A pointer to a single function in to the foreign language.
+    /// Typically, this isn't the callback that's exposed to foreign languages
```

Comment:
nit: this comment feels incomplete - if it's not typically that, then what typically *is* it?